emcc.py

```diff
@@ -1111,6 +1111,9 @@ def check(input_file):
       separate_asm = True
       shared.Settings.FINALIZE_ASM_JS = False
 
+    if shared.Settings.GLOBAL_BASE < 0:
+      shared.Settings.GLOBAL_BASE = 8 # default if nothing else sets it
```

Comment:
If this is specifying a default, then could it directly live as the default value here https://github.com/kripken/emscripten/blob/master/src/settings.js#L75?